src/main/java/somethingrandom/interfaceadapters/additem/AddItemView.java

```diff
@@ -0,0 +1,143 @@
+package somethingrandom.interfaceadapters.additem;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class AddItemView extends JPanel implements ActionListener, PropertyChangeListener {
+
+    private static JPanel panel = new JPanel();
```

Comment:
These shouldn't be `static`: in `main`, you should make an AddItemView (making the controller/viewmodel/etc.), and then add it to a new JFrame.